### DISCUSSION TITLE
on Raspberry 5?

### DISCUSSION THREAD

**Discussion by Rasjel666** [ptitSeb/box86#908] [Q&A]
Hello is it Possible to install this on a Raspberry 5? 

I tried to install it but it doesent work. 
i tryed the -DRPI4ARM64 version of it

**Comment by ptitSeb** [comment]
It's possible, but there is no profile for RPi5 yet. Also, some OS use a 16K pagesize, which needs a special compile flags (and makes most x86 stuff unusable).
So OS keep the 4K pageize, in that case, PI4 build should work.

**Reply by Rasjel666** [reply]
i tryed the PI4 Build but i get a ton of error messages after input the Make command. 
anyone tryed it befor and can help a noob :(

**Comment by purplebar0** [comment]
Late now, but since it doesn't appear to be mentioned anywhere, instead of the default kernel, `kernel8.img` has to be forced (uses 4K page-size, helping avoid segmentation fault).

Edit `/boot/firmware/config.txt` and add this line:
```
kernel=kernel8.img
```

The rest is already described [in docs](https://github.com/ptitSeb/box86/blob/master/docs/COMPILE.md#for-raspberry-pi-on-64bit-os).
